docs/management/index-lifecycle-policies/example-index-lifecycle-policy.asciidoc

```diff
@@ -1,23 +1,167 @@
 [role="xpack"]
+
 [[example-using-index-lifecycle-policy]]
-=== Example of using an index lifecycle policy
+=== Tutorial:  Use ILM to manage Filebeat time-based indices
+
+With index lifecycle management (ILM), you can create policies to define actions
+that are automatically performed against indices as they age.  In this tutorial,
+you will use {kib}’s *Index Lifecycle Policies* to modify and create ILM policies.
+
+
+[float]
+==== Scenario
+
+You’ve been tasked with sending syslog files to an {es} cluster. This
+project has the following data retention guidelines for log data:
+
+* Keep logs on hot data nodes for 30 days
+* Rollover to a new index if the size reaches 50GB
+* After 30 days:
+** Move the logs to warm data nodes
+** Set replicas to 1
+** Force merge indices to a single segment
+* Delete logs after 90 days
+
+
+[float]
+==== Prerequisites
+
+To complete this tutorial, you'll need:
+
+* An {es} cluster with hot and warm nodes configured for shard allocation
+awareness. If you’re using https://www.elastic.co/guide/en/cloud/current/ec-getting-started-templates-hot-warm.html[Elasticsearch Service],
```

Comment:
```suggestion
awareness. If you’re using {cloud}/ec-getting-started-templates-hot-warm.html[Elasticsearch Service],
```

Reply:
I did some more checking into how this works, and I think we should say [`Index-level shard allocation filtering`](https://www.elastic.co/guide/en/elasticsearch/reference/current/shard-allocation-filtering.html) instead of shard allocation awareness.  The Index-level page mentions ILM.